src/5.0_to_5.1/migrateUserActivity.test.ts

```diff
@@ -0,0 +1,23 @@
+import { migrateUserActivity } from "./migrateUserActivity";
+import { contentServer } from "./__test_resources__/contentServer";
+import {
+  dualCatalogDataModel,
+  sandboxDataModel,
+} from "@activeviam/data-model-5.1/dist/__test_resources__";
+
+const dataModels = {
+  sandbox: sandboxDataModel,
+  dualCatalog: dualCatalogDataModel,
+};
+
+describe("migrateUserActivity", () => {
+  it("migrates 5.0 `userFilters` and `userQueryContext` to usable data in 5.1", () => {
+    migrateUserActivity(contentServer, dataModels);
+    expect(
+      contentServer.children?.ui.children?.users.children?.user2.children
+        ?.activity.entry.content,
+    ).toMatchInlineSnapshot(
```

Comment:
I think we should snapshot the parsed object instead, that way the snapshot will be one several lines and easier to read

Reply:
you got it boss